### PR TITLE
upgrade ngraph to ov version

### PR DESCRIPTION
**Pull request by kanvi-nervana**
Upgrade nGraph version in the bridge to the version used by OV

### CODE REVIEW

**Comment by kanvi-nervana** [comment]
Some (13) TF py tests are failing with the error: 
--
  | While validating node 'v0::Add Add_23976034(Constant_23976029[0]:i32{0}, Reshape_23976033[0]:i32{1}) -> (dynamic?)':
  | Argument shapes are inconsistent.

File: CMakeLists.txt [hunk 1]
```diff
@@ -248,7 +248,7 @@ if (NOT USE_PRE_BUILT_NGRAPH)
     ExternalProject_Add(
         ext_ngraph
         GIT_REPOSITORY https://github.com/NervanaSystems/ngraph
-        GIT_TAG v0.28.0-rc.1
+        GIT_TAG 8e37fbfaeb7ee827746d607d9688dc509a25fe77
```
Comment (adk9) [inline CMakeLists.txt:251]: Would it work if you simply specified `tf-bridge-ie-backend` here?

File: bazel/WORKSPACE [hunk 2]
```diff
@@ -61,11 +61,11 @@ tf_workspace(
...
     name = "ngraph",
     build_file = "//:bazel/ngraph.BUILD",
-    sha256 = "24e723f7ed47e2c0068fb5868d8ca88542948a13c9de063d8108f2b67785b089",
-    strip_prefix = "ngraph-0.28.0-rc.1",
+    sha256 = "",
+    strip_prefix = "ngraph-8e37fbfaeb7ee827746d607d9688dc509a25fe77",
     urls = [
-        "https://mirror.bazel.build/github.com/NervanaSystems/ngraph/archive/v0.28.0-rc.1.tar.gz",
-        "https://github.com/NervanaSystems/ngraph/archive/v0.28.0-rc.1.tar.gz",
+        "https://mirror.bazel.build/github.com/NervanaSystems/ngraph/archive/8e37fbfaeb7ee827746d607d9688dc509a25fe77.tar.gz",
```
Comment (adk9) [inline bazel/WORKSPACE:67]: You can use `https://github.com/NervanaSystems/ngraph/archive/tf-bridge-ie-backend.tar.gz` at both places here.

File: bazel/ngraph.BUILD [hunk 3]
```diff
@@ -67,7 +67,7 @@ cc_library(
         "-fstack-protector-all",
         '-D SHARED_LIB_PREFIX=\\"lib\\"',
         '-D SHARED_LIB_SUFFIX=\\".so\\"',
-        '-D NGRAPH_VERSION=\\"v0.28.0-rc.1\\"',
+        '-D NGRAPH_VERSION=\\"8e37fbfaeb7ee827746d607d9688dc509a25fe77\\"',
```
Comment (adk9) [inline bazel/ngraph.BUILD:70]: This could also be simply `tf-bridge-ie-backend` rather than a specific commit?

File: build_ngtf.py [hunk 4]
```diff
@@ -53,7 +53,7 @@ def main():
     '''
 
     # Component versions
-    ngraph_version = "v0.28.0-rc.1"
+    ngraph_version = "8e37fbfaeb7ee827746d607d9688dc509a25fe77"  #adk9 - Upgrade v0::reshape to v1::reshape in opset1
```
Comment (adk9) [inline build_ngtf.py:56]: Change to `tf-bridge-ie-backend` here.

Just to clarify – it'll be much easier for our development and testing to track the (`HEAD` of) `tf-bridge-ie-backend` as we anticipate the target branch to change.

File: CMakeLists.txt (hunk 1 above)
Comment (kanvi-nervana) [reply CMakeLists.txt:251]: Haven't tried

**Pull request closed without merging**